### PR TITLE
tests-invoke: show a bodhi link for the updates-testing scenario

### PR DESCRIPTION
**Pull request by jelly**
When the updates-testing scenario include a link to the bodhi updates page for convenience.

* [ ] Unbreak cockpituous CI `/bin/sh: line 2: update-ca-trust: command not found`

### CODE REVIEW

File: tests-invoke [hunk 1]
```diff
@@ -107,6 +107,12 @@ def main():
                 body += f", [logs]({logs_url}/log.html)"
             if test_scenario != "":
                 title += f"/{test_scenario}"
+
+            assert test_os is not None
+            rel = test_os.split('-')[-1]
```
Comment (martinpitt) [inline tests-invoke:112]: This should be a bit stronger and check if it starts with `fedora-`. Also, let's check for `updates-testing` first, so that none of this runs for other scenarios.

Comment (jelly) [reply tests-invoke]: Correct, adjusted it all.

**Comment by jelly** [comment]
`update-ca-trust` is no longer a part of the minio container which makes our CI not run. Nor does it contain microdnf or dnf so we can't really update any packages.

Can we just append the certificate and try that?

**Comment by jelly** [comment]
Or maybe?

```
${HOME}/.minio/CAs
  myCA.crt
```

**Review by martinpitt — APPROVED** [review]
Let's sort out the cockpituous test separately, you already have a PR for it. Thanks!
